### DISCUSSION TITLE
Telegram的隱憂?!

### DISCUSSION THREAD

**Discussion by 11timyyy** [1101-GS4719-C/Python-Community-GS4719-C#404] [General]
上課時使用的telegram應用程式，近期在烏克蘭也正大量地被使用。烏克蘭政府官員們利用telegram來報導戰況以及俄羅斯的動態等等。
但近期美國通訊軟體 Signal 創辦人 Moxie Marlinspike發表隱憂，Telegram非真加密軟體，且Telegram 的預設訊息會存在於官方伺服器中的雲端資料庫，有權存取資料庫的人都可看見小組中的資訊。也因此Signal推廣他們的通訊都是 e2ee（點對點加密），表示能安全地使用。

文章來源:https://technews.tw/2022/03/01/signals-founder-warns-ukrainians-about-using-telegram/

**Comment by sean1325418** [comment]
如果只是報導的話還好，反正報導就是要讓所有人都知道，只要不拿來傳遞情報就好

**Comment by chupochun** [comment]
希望烏俄戰爭能早點平息

**Comment by chia-kai-chang** [comment]
Telegram預設不是點對點傳送資料，但是在通訊時，可以「選擇」使用點對點傳送資料或是閱後即焚功能。比起LINE來說多了隱私的選擇權利。

參考這篇文章：https://www.techbang.com/posts/90597-privacy-first-these-three-encrypted-communication-softwares

文中提到：

> 綜觀而言，Telegram 在方便性與安全性上取得了折衷，強力的演算法使資料即便儲存於雲端，仍具備比主流通訊軟體更進階的隱私保障，且同時兼顧了跨平台的使用體驗；額外提供之不可備份、轉移的端到端秘密聊天模式，則給擁有更進一步隱私需求的使用者進行選擇。

也就是端對端的秘密聊天是 Telegram提供的選項，供使用者選擇使用。

**Reply by 11timyyy** [reply]
謝謝老師的補充🙏🙏

**Reply by chia-kai-chang** [reply]
很好，這樣的資訊，放Reply即可~ 還可以賺0.3點~ 😂

**Comment by ghost** [comment]
Telegram 也可用聊天機器人，之後得找時間了解一下了....
身邊有人在用挺吸引我的

**Reply by chia-kai-chang** [reply]
小八卦~ 這學期的AI人工智慧導論(A班)會教Telegram聊天機器人~

**Comment by pekoranoinu** [comment]
現在真的是任何事都和科技脫不了關係。

**Comment by chia-kai-chang** [comment]
https://www.blocktempo.com/pavel-durov-said-no-longer-have-any-companies-or-employees-in-russia/

> 「在俄烏開戰的背景下，Telegram 成為當地活躍的通訊軟體，但此前 Signal 創辦人曾質疑 TG 的隱私性。今天，Pavel Durov 發文澄清。他首先表示，自己的媽媽就具有烏克蘭血統，且還有很多親戚住在烏克蘭。所以這場悲劇性的戰爭對他個人和 Telegram 來說都很重要」

**Comment by boson13579** [comment]
!!!握草，只有我現在才知道telegram其實是沒有加密的嗎? 是說messenger最近似乎多出了端對端加密的東西，雖然安全，但是讓應用程式變卡，又好多功能不能用(ex:查看相片，檔案，改暱稱)，總之就變得好麻煩，我跟朋友的屁話是真的不用那麼安全啦!有人知道怎麼取消嗎?
